notebooks/metrics_demo.ipynb

```diff
@@ -72,6 +76,7 @@
     "X_class_train, X_class_test, y_class_train, y_class_test = train_test_split(X_class,\n",
     "                                                                            y_class, train_size=0.8,\n",
     "                                                                            random_state=10)\n",
+    "# Dataset created to be used for the Linear Regression model\n",
     "X_reg, y_reg = make_regression(n_samples=n_samples, n_features=n_features,\n",
     "                               n_informative=n_info, random_state=123)\n",
     "X_reg = X_reg.astype(np.float32)\n",
```

Comment:
Do we need to do astype float32 everywhere? Can't we generate them in float32? Would be a nice example to show users how to avoid unnecessary memory overhead here.